mkdocs.yml

```diff
@@ -119,6 +119,7 @@ nav:
       - Tasks: features/advanced_tasks.md
       - Multi Platform: features/multi_platform_configuration.md
       - Multi Environment: features/multi_environment.md
+      - Pixi Global Manifest: features/pixi_global_manifest.md
```

Comment:
Should we put this under RFCs?

Reply:
I am fine with that. Is that where the `multi_environment` document used to live?

Reply:
Yep! Or proposals, I forgot. @ruben-arts ?

Reply:
Yes, we've previously had a design proposal subsection, util the feature was actually in and it became documentation. So would be good to do that again.

Reply:
Done